src/mjolnir/ingest_transit.cc

```diff
@@ -72,42 +58,42 @@ struct feedObject {
 
 // information referred by tileBuilder saved spatially
 namespace std {
-template <> struct hash<feedObject> {
-  inline size_t operator()(const feedObject& o) const {
+template <> struct hash<feed_object_t> {
+  inline size_t operator()(const feed_object_t& o) const {
     return hash<string>()(o.id + o.feed);
   }
 };
-template <> struct equal_to<feedObject> {
-  inline bool operator()(const feedObject& a, const feedObject& b) const {
+template <> struct equal_to<feed_object_t> {
+  inline bool operator()(const feed_object_t& a, const feed_object_t& b) const {
     return a.id == b.id && a.feed == b.feed;
   }
 };
 } // namespace std
 
 namespace {
 
-struct tileTransitInfo {
-  GraphId graphId;
+struct tile_transit_info_t {
+  GraphId graphid;
   // TODO: unordered multimap-string to a pair of strings that maps from station (parent_id) ->
   //  platform/egress (child) (Max 2 kinds / many could exists)
-  std::unordered_multimap<feedObject, gtfs::Id> tile_station_children;
-  std::unordered_set<feedObject> tile_stops;
-  std::unordered_set<feedObject> tile_trips;
-  std::unordered_map<feedObject, size_t> tile_routes;
-  std::unordered_set<feedObject> tile_services;
-  std::unordered_set<feedObject> tile_agencies;
-  std::unordered_map<feedObject, size_t> tile_shapes;
-
-  bool operator<(const tileTransitInfo& t1) const {
-    // sort tileTransitInfo by size
-    return tile_stops.size() < t1.tile_stops.size();
+  std::unordered_multimap<feed_object_t, gtfs::Id> station_children;
+  std::unordered_set<feed_object_t> stations;
+  std::unordered_set<feed_object_t> trips;
+  std::unordered_map<feed_object_t, size_t> routes;
+  std::unordered_set<feed_object_t> services;
+  std::unordered_set<feed_object_t> agencies;
+  std::unordered_map<feed_object_t, size_t> shapes;
+
+  bool operator<(const tile_transit_info_t& t1) const {
```

Comment:
cool i guess we can delete this function since we dont use it in any ordered containers?

Reply:
the priority queue still needs it

Reply:
strange that code coverage said its not used or maybe I misunderstood the coverage annotations